frontend/src/components/maps/leaflet/Map.tsx

```diff
@@ -317,6 +247,13 @@ const Map: React.FC<MapProps> = ({
     }
   };
 
+  const handleBounds = (e: any) => {
+    const boundsData: LatLngBounds = e.sourceTarget.getBounds();
+    if (!isEqual(boundsData.getNorthEast(), boundsData.getSouthWest())) {
```

Comment:
I think I gotta start reading into more of the lodash capabilities, seem to have a lot of useful methods rather than doing vanilla JS operations